scripts/upload-package.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash -e
+
+#===================================================================================
```

Comment:
This is for dev purposes or CI?

Reply:
The script itself? It's duplicated, very similar in stage and composer. Used in both - CI (system tests) and development.

Or the comment? I tried to find some style guide for bash scripts how to document them. It's not standardized, but that way it looks fine imo.

Reply:
The script :)